introduction/de/03-how-works.asciidoc

```diff
@@ -0,0 +1,47 @@
+== Wie funktioniert InnerSource?
+
+Angenommen, Team A verwendet Software, die von Team B produziert wird.
+Team A sendet eine Funktionsanforderung an Team B, aber Team B kann diese Funktion nicht rechtzeitig für Team A implementieren.
+In einer InnerSource Umgebung kann Team A anstatt einer Feature-Anforderung auch alternativ einen PullRequest an Team B zur Integration schicken.
```

Comment:
```suggestion
Nehmen wir folgendes Szenario an: Team A verwendet Software, die von Team B produziert wird.
Team A übermittelt Anforderungen an Team B, aber Team B ist stark ausgelastet und kann diese nicht rechtzeitig implementieren.
InnerSource ermögicht Team A, anstatt einer Anforderungen, einen PullRequest mit der eigenen Implementierung an Team B zu schicken.
```

Reply:
I havent been involved with the Learning Path lately. Was it conscious decision (in the English version) to use `Pull Request` (Github specific term) vs. something more general like `code contribution`, `change proposal`, ....

Reply:
I feel like this term is universally-understood and the clearest way to express what we're talking about.  If I saw one of those other terms I feel like people would want to ask something like `You mean a pull request?`.  _BitBucket_ and _GitLab_ both mention pull requests as well.

Reply:
I'm willing to be proved wrong, though, if there's reason to believe that the term "pull request" might be confusing to some.

Reply:
All git systems I know use pull request as the term. "git system" may be the limitation here, but I think today it is safe to say they are vastly dominating for version control, no?  Pull request is really the mechanics, the term "sending a pull request" however I think at this point can be used exchangeable with sending a code contribution. (just like paper handkerchief and Kleenex)..